backups/salt-backup/salt-backup.py

```diff
@@ -16,25 +19,35 @@
 with open(CONFIG_FILE, 'r') as f:
     config = yaml.load(f, Loader=yaml.FullLoader)
 
-
 print("Running salt backup")
-command = "b2 sync /etc/salt b2://salt-backup/salt"
-result = subprocess.run(command, shell=True)
 
-if result.returncode == 0:
-    success = 1
-    print("salt backup success")
-else:
-    success = 0
-    print("salt backup failed")
-    sys.exit()
+# Make zip of files to backup
+dirs = ["/etc/salt", "/srv/pillar"]
+temp_dir = tempfile.mkdtemp()
+for dir in dirs:
+    shutil.copytree(dir, os.path.join(temp_dir, os.path.basename(dir)))
+zip_file = f"{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}_salt-backup.zip"
+shutil.make_archive(zip_file.replace(".zip", ""), 'zip', temp_dir)
```

Comment:
Should we delete the zipped directory?

Reply:
why are we replacing the zip extension?, just dont create is a .zip in the line prior if we don't want .zip

Reply:
`shutil.make_archive` will add `.zip` to the filename given, even if it already ends in `.zip`, so I just remove it in that one place instead of adding `.zip` is all other places `zip_file` is used.